gnd/src/main/java/com/google/android/gnd/util/ImmutableListCollector.java

```diff
@@ -37,11 +38,37 @@ private ImmutableListCollector() {}
           },
           ImmutableList.Builder::build);
 
+  /** Do not instantiate. */
+  private ImmutableListCollector() {}
+
   /**
    * Returns a {@link Collector} that accumulates the input elements into a new ImmutableList, in
    * encounter order.
    */
   public static <E> Collector<E, ?, ImmutableList<E>> toImmutableList() {
     return (Collector) TO_IMMUTABLE_LIST;
   }
+
+  /**
+   * Returns a function for use in rx streams that maps a function over the contents of an immutable
```

Comment:
rx streams - > RxJava observables here and below for consistency with our docs.

Reply:
Done!